src/EnergyPlus/PsychCacheData.hh

```diff
@@ -100,7 +100,7 @@ constexpr Int64 psatcache_mask = psatcache_size - 1;
 #endif
 #ifdef EP_cache_PsyTsatFnPb
 constexpr int tsatcache_size = 1024 * 1024;
-constexpr int tsatprecision_bits = 24;
+constexpr int tsatprecision_bits = 20;
```

Comment:
I think if convTol is set to 0.01C, this can go down to 16.

Reply:
I asked @LipingWang to look at this earlier and we found that while it increases speed slightly to go from 20 to 16, it also increases the differences in energy, so we decided to stick with 20. This was on a small set of buildings and we can evaluate that with a larger set of buildings to get a more accurate picture if you want.